City-of-Helsinki/drupal-hdbt-admin
# UHF-5897 News item promote

# [UHF-5897](https://helsinkisolutionoffice.atlassian.net/browse/UHF-5897)

## What was done
* Added form alter for the promoted to front page checkbox and styles for it.

## How to install
* **NOTE!** This branch should be tested in [Etusivu instance](https://github.com/City-of-Helsinki/drupal-helfi-etusivu/pull/54)
* Make sure the etusivu instance is up and running on latest dev branch.
    * `git pull origin UHF-5897_news_item_promote`
    * `make fresh` or `make new`
* Update the HDBT Admin theme
    * `composer require drupal/hdbt_admin:dev-UHF-5897_news_item_promote`
* Run `make drush-cr`
* Open shell `make shell` and run `drush locale:check; drush locale:update; drush cr` to update the translations

## How to test
* Go to add or edit news item (f.e https://helfi-etusivu.docker.so/fi/node/add/news_item) 
* Check that the **Promoted to front page** has been converted to **Publish the news item in the top news flow** checkbox and it has moved to more prominent part of the sidebar. See screenshots:
   
   * ![image](https://user-images.githubusercontent.com/1712902/171841437-c825521c-d80e-460c-9f91-641bdcc2df34.png)

   * ![image](https://user-images.githubusercontent.com/1712902/171841386-1e5c9e5a-dbdb-45a4-a1b7-8095b0edf8f8.png)

## Other PRs
* https://github.com/City-of-Helsinki/drupal-helfi-etusivu/pull/54
